src/lib/plugin_apis/part.api

```diff
@@ -49,7 +49,8 @@ typedef enum {
     BD_PART_FLAG_GPT_SYSTEM_PART = 1 << 25,
     BD_PART_FLAG_GPT_READ_ONLY = 1 << 26,
     BD_PART_FLAG_GPT_HIDDEN = 1 << 27,
-    BD_PART_FLAG_GPT_NO_AUTOMOUNT = 1 << 28
+    BD_PART_FLAG_GPT_NO_AUTOMOUNT = 1 << 28,
+    BD_PART_FLAG_GPT_LEGACY_BOOT = 1 << 29
```

Comment:
The flag is already defined above as ``BD_PART_FLAG_LEGACY_BOOT``. It's one of the flags supported by *libparted*.